flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -220,6 +274,12 @@ def __exit__(self, type, value, traceback):
 
 
 class ISO8601Date(confuse.Template):
+    """
+    Reads in config dates into datetimes.dates.
+
+    Attributes:
+        value: Date value.
```

Comment:
I don't think this class assigns a value to `self.value`, I think the "Attributes" section here can be removed.